prov/cxi/include/cxip.h

```diff
@@ -177,7 +177,7 @@
 #define CXIP_MINOR_VERSION		1
 #define CXIP_PROV_VERSION		FI_VERSION(CXIP_MAJOR_VERSION, \
 						   CXIP_MINOR_VERSION)
-#define CXIP_FI_VERSION			FI_VERSION(1, 21)
+#define CXIP_FI_VERSION			FI_VERSION(2, 0)
```

Comment:
This seems like a separate change

Reply:
That has already been addressed as part of HPE's latest updates to the CXI provider